src/main/resources/static/css/login.css

```
@@ -0,0 +1,71 @@
+@media (min-width: 640px) {
+        .label-container {font-size: 24px;}
+        .input-field {min-width: 220px;}
+        .form-container {gap: 1.25rem;}
+        .invalid-state {font-size: 14px;}
+        .warning-state {font-size: 14px;}
+        .login-button {font-size: 20px;}
+	}
+
+@media (max-width: 640px) {
+        .label-container {font-size: 14px;}
+        .input-field {min-width: 150px;}
+        .form-container {gap: 0.75rem;}
+        .invalid-state {font-size: 10px;}
+        .warning-state {font-size: 10px;}
+        .login-button {font-size: 14px;}
+    }
+
+.login-container {
+    box-sizing: border-box;
+    height: max-content;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+}
+
+.form-container {
+    display: inline-flex;
+    flex-direction: column;
+}
+
+.label-container {
+    display: inline-flex;
+    flex-direction: column;
+}
+
+.login-button {
+    margin-left: auto;
+}
+
+.login-button > input {
+    font-weight: 400;
+	text-align: center;
+	border: 1px solid transparent;
+	padding: .375rem .75rem;
+	line-height: 1.5;
+	border-radius: .25rem;
+	color: #fff;
+	background-color: #007bff;
+}
+
+.input-field {
+    border: 1px solid black;
+    border-radius: .25rem;
+}
+
+.waves {
+    position: fixed;
+    bottom: 0;
+    width: 100%;
+    z-index: -1;
+}
+
+.invalid-state {
+    color: crimson;
+}
+
+.warning-state {
+    color: grey;
+}
```

Exactly one blank line at end of file
```suggestion
}
```

---

I've added the blank line!